### PR TITLE
47376_supervisor_view_supervisors

### PR DESCRIPTION
**Pull request by yasin-buet**
supervisor view supervisors.selecting a supervisor, one can see his/her courses
added migration for supervisor_id column in course table.
screenshot from 2015-12-08 12 12 07
screenshot from 2015-12-08 12 47 38
screenshot from 2015-12-08 12 48 09

### CODE REVIEW

**Comment by asifbuetcse** [comment]
ok

File: resources/views/supervisor/users/supervisors/show.blade.php [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+            <thead>
+                <tr>
+                    <th>Course Name</th>
+                    <th>Course Descripion</th>
+                </tr>
+            </thead>
+            <tbody>
+                @foreach ($courses as $course)
+                    <tr>
+                        <td>{{ $course->name }}</td>
```
Comment (dinhhoanglong91) [inline resources/views/supervisor/users/supervisors/show.blade.php]: Please add link to course detail page.
Please show supervisor name in this page.

File: app/Http/Controllers/Supervisor/UsersController.php [hunk 2]
```diff
@@ -51,8 +56,10 @@ public function store(Request $request)
      * @return \Illuminate\Http\Response
      */
     public function show($id)
-    {
-        //
+    {  
+        $supervisor = User::find($id);
+        $courses = Course::whereSupervisorId($id)->get();
```
Comment (dinhhoanglong91) [inline app/Http/Controllers/Supervisor/UsersController.php]: We need something like this
https://github.com/framgia/tms_42/pull/13/files#diff-1f16bbfd6b5abb4b8d48041fc0e83808R54

**Comment by dinhhoanglong91** [comment]
ok

**Pull request merged**